src/app/app-routing.module.ts

```diff
@@ -788,12 +789,12 @@ const routes: Routes = [
     loadChildren: () => import('./home/home.module').then(m => m.HomeModule),
     data: {
       pageType: 'page',
-      pageKey: 'home',
+      pageKey: 'homee',
```

Comment:
Please change the key to home only , spelling mistake

Reply:
done